src/events/filesink.rs

```diff
@@ -0,0 +1,112 @@
+use async_std::fs::OpenOptions;
```

Comment:
Actually, now that I am looking at this again, I think all of these options are available in tokio (including the prelude below)